packages/eslint-plugin-pf-codemods/src/rules/v6/userFeedbackWarnChanges/user-feedback-warn-changes.ts

```diff
@@ -0,0 +1,36 @@
+import { getFromPackage } from "../../helpers";
+import { Rule } from "eslint";
+import { ImportDeclaration } from "estree-jsx";
+
+// https://github.com/patternfly/react-user-feedback/pull/76
+module.exports = {
+  meta: {},
+  create: function (context: Rule.RuleContext) {
+    const { imports } = getFromPackage(context, "@patternfly/react-user-feedback");
```

Comment:
It looks like user feedback supports default exports, so I think we probably need to check for those here too.

For this you should be able to just update your rule here to use the `getAllImportsFromPackage` helper instead of the `getFromPackage` helper, and then the `getDefaultDeclarationString` helper to handle those default imports. If you have any questions on that just lmk.

Probably also want to add a test for that.

Reply:
Additionally I don't think this would catch any imports from `react-user-feedback/dist`, for that I think the quickest path would be to update `pfPackageMatches` to look for anything in user feedback like it does component groups.

And we should a test for that too.

Reply:
I'm not sure what you mean by updating `pfPackageMatches`, that seems to be used in place of `getAllImportsFromPackage` which I'm currently using. Should I swap/adjust the import? 

I've also tried to adjust to account for the default import but my added (commented out) test isn't working - it's not returning anything from the `getAllImportsFromPackage` helper in the rule so no errors are reported.

@wise-king-sullyman